tests/Spectra.py

```diff
@@ -48,7 +48,7 @@ def setUp(self):
         
         self.nFiberTraces = 11
         self.nRowsPrescan = 49
-        self.maxRMS = 0.053
+        self.maxRMS = 0.067
```

Comment:
If I've understood the commit message, then this number seems to be pretty poorly justified: it's just a value you happened to measure one day. I guess that's better than nothing (at the least, it serves as a regression test to demonstrate that changes to the code elsewhere haven't made it worse), but you should add a comment to that effect.